addons/account/models/account_account_tag.py

```diff
@@ -14,6 +14,15 @@ class AccountAccountTag(models.Model):
     tax_negate = fields.Boolean(string="Negate Tax Balance", help="Check this box to negate the absolute value of the balance of the lines associated with this tag in tax report computation.")
     country_id = fields.Many2one(string="Country", comodel_name='res.country', help="Country for which this tag is available, when applied on taxes.")
 
+    _sql_constraints = [('name_uniq', "", "Tag name already exist!")]
+
+    def init(self):
+        self._cr.execute("""
+            CREATE UNIQUE INDEX IF NOT EXISTS account_account_tag_name_uniq
+            ON account_account_tag (name)
+            WHERE applicability = 'accounts'
+        """)
```

Comment:
```suggestion
    _sql_constraints = [('name_uniq', "unique(name, applicability, country_id)", "A tag with the same name and applicability already exists in this country.")]
```
I think we could simply do this :wink:

Reply:
no, it's different because I want only the tags with applicability : account to be unique, this would make all tax aplicability country_id pairs unique they are different

Reply:
I know it's different (which is why I changed the error message) :wink: 
I just wanted to challenge the idea of having a constraint for each applicability, not just accounts :slightly_smiling_face:

Reply:
yes I understand, but that is up to the PO I guess, I he agrees i'd gladly do it !

Reply:
ping @epictete

Reply:
Since it's your task, I thought you were going to check with the PO :wink: 
But anyways, I just checked with OCO and OLMA and they both agreed :ok_hand: 
(don't forget to update the upgrade PR accordingly :slightly_smiling_face:)

Reply:
arighty !
Thanks